src/main/java/edu/kis/powp/jobs2d/drivers/adapter/DrawPanelAdapter.java

```diff
@@ -26,8 +29,10 @@ public void operateTo(int x, int y) {
 		ILine line = LineFactory.getBasicLine();
 		line.setStartCoordinates(this.startX, this.startY);
 		line.setEndCoordinates(x, y);
+		setPosition(x, y);
 
-		drawLine(line);
+		DrawPanelController drawPanelController = getDrawerController();
+		drawPanelController.drawLine(line);
```

Comment:
Constructor injection should be used (using a parameter). This is unnecessary direct coupling with DrawerFeature class.